drivers/ce_performance/app/models/ce_performance/result.rb

```diff
@@ -172,5 +181,29 @@ def data_for_events(report)
         end
       end
     end
+
+    def data_for_exits(report)
+      @data_for_exits ||= {}.tap do |data|
+        self.class.available_exit_destination_ids.each do |event_id|
```

Comment:
I'm thinking these should be `destination` instead of `event_id`
```suggestion
        self.class.available_exit_destination_ids.each do |destination|
```
There are move uses of it further down too.